src/components/footer/footer.style.js

```diff
@@ -0,0 +1,41 @@
+import styled, { css } from 'styled-components/macro';
+import { Link as ReactRouterLink } from 'react-router-dom';
+import { ReactComponent as SignLogo } from '../../img/sign.svg';
+
+export const Container = styled.footer`
+  height: 100px;
+  width: 100vw;
+  display: flex;
+`;
+
+export const Frame = styled.div`
+  width: 100%;
+  display: flex;
+  align-items: center;
+  max-width: 940px;
+  margin: auto;
+  
+  @media(max-width: 940px) {
+      padding: 0 20px;
+  }
+`;
+
+export const Sign = styled(SignLogo)`
+  display: block;
+`;
+
+const baseTextLink = css`
+  color: ${(props) => props.theme.color.gray};
+  font-size: 0.875rem;
+`;
+
+export const TextRouterLink = styled(ReactRouterLink)`
+  flex:1;
```

Comment:
The disadvantage of setting `flex: 1` to the link itself is that the user might click into the empty space between the link and the logo and be surprised to have clicked a link.

Alternatively, you could wrap the link inside a `div` or so and set `flex: 1` there. It's not a big deal though tbh 😄